language/de-DE/com_users.ini

```
@@ -42,27 +42,58 @@ COM_USERS_LOGIN_USERNAME_LABEL="Benutzername"
 COM_USERS_MAIL_FAILED="E-Mail-Versand fehlgeschlagen."
 COM_USERS_MAIL_SEND_FAILURE_BODY="Beim Versenden der Registrierungsmail ist ein Fehler aufgetreten. Folgender Benutzer wollte sich registrieren: %s"
 COM_USERS_MAIL_SEND_FAILURE_SUBJECT="Fehler beim Versand der E-Mail"
+COM_USERS_MFA_ADD_AUTHENTICATOR_OF_TYPE="Hinzufügen einer neuen %s"
```

```suggestion
COM_USERS_MFA_ADD_AUTHENTICATOR_OF_TYPE="%s hinzufügen"
```

Wäre eine Variante für alle 3 Fälle.

---

Ja würde ich so nehmen